.github/workflows/build-no-os.yaml

```diff
@@ -0,0 +1,59 @@
+name: no-OS
```

Comment:
@pamolloy 
1. I think name of this file should be build-projects.yaml. We are not building any no-OS projects here
2. name of pipeline should be name: precision-converters-firmware